include/union_room.h

```diff
@@ -4,28 +4,34 @@
 #include "global.h"
 #include "link_rfu.h"
 
-struct UnkStruct_Shared
+// Return value of IsRequestedTypeAndSpeciesInPlayerParty
+#define UR_TRADE_MATCH  0
+#define UR_TRADE_NOTYPE 1
+#define UR_TRADE_NOEGG  2
+
+#define UROOM_MAX_GROUP_COUNT 8
+#define UROOM_MAX_PARTY_SIZE  5
```

Comment:
Is this used? I associate party size with pokemon party size, and I havent heard of a union room pokemon party limit